docs/Concepts/Transactions/Transaction-Pool.md

```diff
@@ -1,5 +1,7 @@
 # Transaction Pool 
 
+Mining nodes maintain a transaction pool where pending transactions are stored before they are processed.
```

Comment:
Note that all nodes contain a transaction pool, not just mining nodes.  It's an important part of ensuring transactions are propagated around the network.

Reply:
Thanks for clearing that up for me! Changed the text to 'all nodes' rather than just 'mining nodes'.